src/components/itemHelper.js

```diff
@@ -148,11 +144,7 @@ export function canEditImages (user, item) {
     }
 
     if (itemType === 'UserView') {
-        if (user.Policy.IsAdministrator) {
-            return true;
-        }
-
-        return false;
+        return user.Policy.IsAdministrator;
```

Comment:
Maybe cast to boolean in case of missing `IsAdministrator`?
```suggestion
        return !!user.Policy.IsAdministrator;
```